src/main/java/org/jenkinsci/test/acceptance/po/UpdateCenter.java

```diff
@@ -71,7 +70,8 @@ public boolean isInstalled(String pluginShortName) throws InstallationFailedExce
      */
     public boolean waitForInstallationToComplete(final PluginSpec... specs) throws InstallationFailedException {
         open();
-        waitFor(driver, not(Matchers.hasContent("Pending")), 60); // Wait for all plugins to get installed
+        // Wait for all plugins to get installed
+        waitFor(driver, not(Matchers.hasElement(by.xpath("//*[@id='log']//*[contains(.,'Pending') or contains(.,'Installing')]"))), 60);
```

Comment:
The 'Installing' string also appears in the title of the page, that's why we need to be more precise, and look for it only inside the "log".